las_geoh5/export_las.py

```diff
@@ -0,0 +1,209 @@
+#  Copyright (c) 2023 Mira Geoscience Ltd.
+#
+#  This file is part of my_app project.

```

Comment:
find and replace `my_app` by  `las-geoh5`

Reply:
done